packages/fluent/scss/progressbar/_variables.scss

```diff
@@ -0,0 +1,76 @@
+@use "../core/color-system" as *;
+@use "../core/_variables" as *;
+
+/// Height of the progressbar.
+/// @group progressbar
+$kendo-progressbar-height: 2px !default;
+/// Horizontal width of the progressbar.
+/// @group progressbar
+$kendo-progressbar-horizontal-width: 100% !default;
+/// Animation timing of the progressbar.
+/// @group progressbar
+$kendo-progressbar-animation-timing: 1s linear infinite !default;
+/// Border width of the progressbar.
+/// @group progressbar
+$kendo-progressbar-border-width: 0px !default;
+/// Border radius of the progressbar.
+/// @group progressbar
+$kendo-progressbar-border-radius: var( --kendo-border-radius-sm, initial ) !default;
+
+/// Font family of the progressbar.
+/// @group progressbar
+$kendo-progressbar-font-family: var( --kendo-font-family, inherit ) !default;
+/// Font size of the progressbar.
+/// @group progressbar
+$kendo-progressbar-font-size: var( --kendo-font-size-sm, inherit ) !default;
+/// Line height of the progressbar.
+/// @group progressbar
+$kendo-progressbar-line-height: var( --kendo-line-height, inherit ) !default;
+
+///  of the progressbar.
+/// @group progressbar
+$kendo-progressbar-offset-y: map-get( $kendo-spacing, 2 ) !default;
+///  of the progressbar.
+/// @group progressbar
+$kendo-progressbar-padding-x: 0 !default;
+///  of the progressbar.
+/// @group progressbar
+$kendo-progressbar-padding-y: 0 !default;
+
+/// Background color of the progressbar.
+/// @group progressbar
+$kendo-progressbar-bg: get-theme-color-var( neutral-30 ) !default;
+/// Text color of the progressbar.
+/// @group progressbar
+$kendo-progressbar-text: var( --kendo-component-text, initial ) !default;
+/// Border color of the progressbar.
+/// @group progressbar
+$kendo-progressbar-border: $kendo-progressbar-bg !default;
```

Comment:
use the CSS variable here with a fallback to the Sass var:

var( --kendo-progressbar-bg, $kendo-progressbar-bg) 

In this way if anyone uses --kendo-progressbar-bg to override the bg color, it'll apply on the border as well.

The same goes for the rest of the sass vars used here